### ISSUE TITLE
wrong info on correspSearch website

### ISSUE THREAD

**Issue by yurj** [correspSearch/CMIF-Creator#25]
Hi!

 on the correspSearch website: https://correspsearch.net/en/participate.html there's a box with:
"CSV to CMIF with Python script

Klaus Rettinghaus has developed a Python script that converts CSV tables into CMIF files. The script and the manual can be found on the [GitHub account of the SAW-Leipzig](https://github.com/saw-leipzig/csv2cmi).

SAW Leipzig now also offers a corresponding webapp that enables the transformation online. [[To the CSV2CMI webapp]](https://cmif.saw-leipzig.de/)"

but Klaus Rettinghaus in this issue: https://github.com/saw-leipzig/csv2cmi/issues/65 says:
"This project is not connected to https://github.com/correspSearch and the generated XML files are not suitable for use in the CMIF-Creator."

What are the tools and the supported CMIF format suited to upload correspondences in correspSearch? Is CMIF the only available format?

Thanks!

**Comment by StefanDumont** [comment]
CMIF files from the CSV2CMI tool can of course be processed by correspSearch, but currently not by the CMIF Creator, because CSV2CMI adds some technical information (i.e. attributes `@xml:id`) that is not supported in the CMIF Creator. However, the CMIF Creator is basically only intended to create or extend a CMIF file. If you want to provide an already existing CMIF file for correspSearch, you only have to provide it online (on any webspace or in an GitHub repo) and send us the URL. Just send us an e-mail at correspsearch@bbaw.de and we will be happy to register the CMIF file or help you in any other way.

**Comment by yurj** [comment]
I'm trying to create a workflow. We have data basically in csv files managed outside and we'll have to create the CMIF. If csv format is not accepted by correspSearch, I think I need some library to create and validate them, are there any?

**Comment by StefanDumont** [comment]
Yes, correspSearch accepts only the CMIF format, i.e. XML. The CSV2CMI tool, maintained by @rettinghaus, (or the webapp, maintained by @saw-leipzig) is usually a good way to convert CSV data to a CMIF XML file. The resulted CMIF file can be validated (with proper XML tools, e.g. Oxygen XML) against the RNG-Schema, which you can find here: https://github.com/TEI-Correspondence-SIG/CMIF/blob/main/schema/cmi-customization.rng (the CMIF documentation can be found [here](https://github.com/TEI-Correspondence-SIG/CMIF/blob/main/doc/documentation-en.md) as well as on our [webpage](https://correspsearch.net/en/documentation.html)). One crucial point in CMIF is that persons should be referenced with VIAF/GND-URIs (and places with GeoNames-URIs). Feel free to ask further questions or just send us a first draft of your CMIF file, than we can provide feedback :-)

**Comment by yurj** [comment]
Thanks! I cannot find the csv format documentation.

 We didn't consider the webapp because returned errors on some test files. I get a 500 on https://cmif.saw-leipzig.de/api/upload when uploading a csv (no hint why the error happen, can be a validation error), but it works with https://github.com/saw-leipzig/csv2cmi/tree/main/examples/shorttitle-replacement (where gnid and geonames does not use urls)

There's some detail I'm missing on the csv file, I just found we used the ' character in the edition title :-/ now it works also with our csv, great.

Thanks for all the precious feedback!

**Comment by rettinghaus** [comment]
@yurj @StefanDumont I suppose this can be closed now?

**Comment by yurj** [comment]
yes, thanks!
